### DISCUSSION TITLE
Baseline in a closed polygon by the Douglas-Peucker algorithm

### DISCUSSION THREAD

**Discussion by Iv-dot** [OSGeo/grass#4467] [Q&A]
Hello,

How is the baseline determined if it is a closed polygon when simplifying with the Douglas-Peucker algorithm? With a regular line, it is the first and last point, but what about a closed polygon (where are the first and last points the same)? 
thanks

**Comment by petrasovaa** [comment]
I don't know but you can try to figure it out from the [source code](https://github.com/OSGeo/grass/blob/main/vector/v.generalize/simplification.c#L27) or experiment with some artificial data.
